Library/Formula/librdkafka.rb

```diff
@@ -0,0 +1,12 @@
+class Librdkafka < Formula
+  homepage "https://github.com/edenhill/librdkafka"
+  url "http://github.com/edenhill/librdkafka/tarball/0.8.4"
```

Comment:
Why not ` https://github.com/edenhill/librdkafka/archive/0.8.4.tar.gz ` out of curiosity?